### DISCUSSION TITLE
Behaviour of `--cache-refresh` when rendering to multiple formats

### DISCUSSION THREAD

**Discussion by sebffischer** [quarto-dev/quarto-cli#7503] [Feature Requests]
### Description

I have noticed that when rendering to multiple output formats (in our case html and pdf) with the option `--cache-refresh` every quarto document is rendered twice. It seems to me, that within one rendering call it makes little sense to re-compute anything (even if some chunks e.g. set  `#| cache: false`) as one probably just wants exactly the same output for both formats.

My suggested feature is to have a command-line option that ensures that the for the first format the cache is cleared before rendering, but for every other format no chunks are re-computed (even overwriting the `#| cache: false` options of chunks). 
(I don't really know how quarto works internally, so my usage of words might be imprecise).

Also, thanks a lot for all your work and efforts, I really enjoy working with quarto! :)

**Comment by cscheid** [comment]
> It seems to me, that within one rendering call it makes little sense to re-compute anything (even if some chunks e.g. set #| cache: false) as one probably just wants exactly the same output for both formats.

That's unfortunately not true; graphics devices can behave differently on different formats (for example).
